manual/mapper/daos/delete/README.md

```diff
@@ -105,6 +105,13 @@ The method can return:
     ResultSet deleteIfDescriptionMatches(UUID productId, String expectedDescription);
     // if the condition fails, the result set will contain columns '[applied]' and 'description'
     ```
+  
+* a [BoundStatement]. This is intended for queries where you will execute this statement later or in a batch.
```

Comment:
Nit: our coding convention uses a column limit of 100 characters, could you add a line break here?